client/src/components/HomePage.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import CategoriesList from "./CategoriesList";
+
+const HomePage = (props) => {
+  return (
+    <>
+      <h2>Hello from Pastabilities Home Page</h2>
+      <CategoriesList />
+    </>
```

Comment:
I'll allow this for now, but this text should be replaced by "Pastabilities" or whatever formal header text you want